observability/opentelemetry/src/main/resources/schema/wildfly-opentelemetry_1_0.xsd

```diff
@@ -0,0 +1,66 @@
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="urn:wildfly:opentelemetry:1.0"
+           xmlns="urn:wildfly:opentelemetry:1.0"
+           elementFormDefault="qualified"
+           version="1.0">
+
+    <xs:element name="subsystem">
+        <xs:complexType>
+            <xs:sequence >
+                <xs:element name="exporter" minOccurs="0" maxOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute name="type" default="jaeger">
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:enumeration value="jaeger"/>
+                                    <xs:enumeration value="otlp"/>
+                                </xs:restriction>
+                            </xs:simpleType>
+                        </xs:attribute>
+                        <xs:attribute name="endpoint" type="xs:string"/>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="span-processor" minOccurs="0" maxOccurs="1">
+                    <xs:complexType>
+                        <xs:attribute type="xs:string" name="type" default="batch"/>
+                        <xs:attribute type="xs:integer" name="batch-delay" default="5000"/>
+                        <xs:attribute type="xs:integer" name="max-queue-size" default="2048"/>
+                        <xs:attribute type="xs:integer" name="max-export-batch-size" default="512"/>
+                        <xs:attribute type="xs:integer" name="export-timeout" default="30000"/>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="sampler" minOccurs="0" maxOccurs="1">
+                    <xs:complexType>
+                                <xs:attribute type="xs:string" name="type"/>
```

Comment:
Why isn't this restricted? Can there be custom values?

What is the default value?

Reply:
Restrictions added.